apps/api/src/app/auth/services/passport/jwt.strategy.ts

```diff
@@ -19,6 +20,12 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
       throw new UnauthorizedException();
     }
 
+    this.logger.assign({
+      userId: user._id,
+      environmentId: payload.environmentId,
+      organizationId: payload.organizationId,
+    });
```

Comment:
Does that mean that this data will be propagated to all of the logs related to this request? do you think that we should add data related to running the machine as well?

Reply:
Data for machine is auto added by newrelic, and yes this will be added to all logs of the request